recipes/vscode-jupyter/build.sh

```diff
@@ -0,0 +1,11 @@
+EXTENSIONS_DIR="${PREFIX}/share/code-server/extensions"
+
+npm install -g vsce
+npm ci
+npm run clean
+npm run package
+
+mkdir -p "${EXTENSIONS_DIR}"
+code-server \
+    --install-extension "ms-toolsai-jupyter-insiders.vsix" \
+    --extensions-dir "${EXTENSIONS_DIR}"
```

Comment:
We don't want to install the extension globally, which is why we set the `--extensions-dir` argument here.
The caveat though is that if you want to use the install extensions inside your virtual environment, you'll have to specify `--extensions-dir` like this:

```sh
$ code-server --extensions-dir $CONDA_PREFIX/share/code-server/extensions
```

Reply:
https://github.com/conda-forge/code-server-feedstock/pull/40